core/src/banking_stage.rs

```diff
@@ -701,6 +701,11 @@ impl BankingStage {
             txs.len(),
         );
 
+        debug!(
```

Comment:
I wasn't sure where to report all of these. Stephen has preferred debug! in the past, so I assumed these would be available if someone wants to look for them.

Reply:
I think [this](https://github.com/solana-labs/solana/blob/830be855dc3c38fe005ba07cf19d71d31eb0f24d/core/src/progress_map.rs#L64) may be a place to add some or all of the new metrics.

Reply:
Yes, in here will be too noisy to turn on by default. Generally we just want to print things at a rate of 1-2 seconds.